src/main/java/teammates/ui/template/InstructorFeedbackResultsSectionPanel.java

```diff
@@ -0,0 +1,146 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class InstructorFeedbackResultsSectionPanel {
+    private String panelClass;
+    private boolean isGroupedByTeam;
+    private boolean isDisplayingMissingParticipants;
+    private boolean isLoadSectionResponsesByAjax;
+    
+    private String sectionName;
+    private String sectionNameForDisplay;
+    
+    private String statisticsHeaderText;
+    private String detailedResponsesHeaderText;
+    
+    // A mapping from team name to a list of participant panels. Each participant panel is for one member of the team
+    private Map<String, List<InstructorResultsParticipantPanel>> participantPanels;
+    
+    private boolean isDisplayingTeamStatistics;
+    private Map<String, List<InstructorResultsQuestionTable>> teamStatisticsTable;
+    private Map<String, Boolean> isTeamWithResponses;
+
+    public InstructorFeedbackResultsSectionPanel() {
+        panelClass = "panel-success";
+        isDisplayingMissingParticipants = true;
+        
+        isTeamWithResponses = new HashMap<String, Boolean>();
+        participantPanels = new LinkedHashMap<String, List<InstructorResultsParticipantPanel>>();
+    }
+    
+    
+    public String getPanelClass() {
+        return panelClass;
+    }
+
+    public void setPanelClass(String panelClass) {
+        this.panelClass = panelClass;
+    }
+
+    public boolean isGroupedByTeam() {
+        return isGroupedByTeam;
+    }
+
+    public void setGroupedByTeam(boolean isGroupedByTeam) {
+        this.isGroupedByTeam = isGroupedByTeam;
+    }
+
+    public String getSectionName() {
+        return sectionName;
+    }
+
+    public void setSectionName(String sectionName) {
+        this.sectionName = sectionName;
+    }
+
+    public Map<String, List<InstructorResultsParticipantPanel>> getParticipantPanels() {
+        return participantPanels;
+    }
+
+    public void setParticipantPanels(
+                                    Map<String, List<InstructorResultsParticipantPanel>> participantPanels) {
```

Comment:
A number of the setters aren't used, will you be cleaning them up at the end and just keeping them around for now?